src/notional/orm.py

```diff
@@ -215,19 +218,27 @@ def icon(self):
         return self._notional__page.icon
 
     @icon.setter
-    def icon(self, emoji):
+    def icon(self, icon: Union[str, EmojiObject, ExternalFile]):
         """Set the icon for the Page.
 
-        :param emoji: may be either a single emoji string or an `EmojiObject`
+        :param icon: may be either a single emoji string or an `EmojiObject`
         """
 
-        if isinstance(emoji, str):
-            emoji = EmojiObject[emoji]
-
-        elif not isinstance(emoji, EmojiObject):
-            raise ValueError("Invalid emoji; unsupported type")
+        if isinstance(icon, str):
+            if icon.startswith(":"):
+                icon = emojize(icon, language='alias')
```

Comment:
Prefer double quotes for style consistency...  Black should take care of this for you if you use the `pre-commit` hooks.

Reply:
My bad, forgot to run `pre-commit install` in my new environment. Fixed this.